.travis.yml

```diff
@@ -18,7 +21,6 @@ install:
   - python3 -m pip install --prefer-binary -r dev_requirements.txt -r requirements.txt
   - mkdir user
   - cp ./config/default_config.json ./user/config.json
-#  - python3 start.py -p install all
```

Comment:
should we remove it ?

Reply:
We should adapt it when we'll migrate the tests I think